async function run() {
	try {
	} catch (err) {
		console.log(err, "err")
	}
}

// @ts-ignore
await run()
// past:
